flexmeasures/ui/templates/views/sensors.html

```diff
@@ -33,6 +33,58 @@
               <div class="alert alert-info" id="dstwarn" style="display:none;"></div>
               <div class="alert alert-info" id="sourcewarn" style="display:none;"></div>
           </div>
+          <div class="container">
+          <div class="row">
+          <div class="col-sm-6">
+                    <h3>Stats</h3>
+                    <table id="statsTable" class="table table-striped">
+                        <tbody id="statsTableBody">
+                        </tbody>
+                    </table>
+                        <script>
+                            fetch('/api/v3_0/sensors/' + {{ sensor.id }} + '/stats')
+                                .then(response => response.json())
+                                .then(data => {
+                                    const tableBody = document.querySelector('#statsTableBody');
+                                    const keys = [
+                                        "min_event_start",
+                                        "max_event_start",
+                                        "min_value",
+                                        "max_value",
+                                        "mean_value",
+                                        "sum_values",
+                                        "count_values"
+                                    ];
+
+                                    for (const sourceKey in data) {
+                                        const stats = data[sourceKey];
+
+                                        keys.forEach(key => {
```

Comment:
I feel like we don't need the `keys` list, and we can simply do `stats.forEach(key => {`, and leave out the if statement in line 63.

Reply:
I did try it with the keys list but because of the sorting the keys as a problem.

Reply:
Okay, let's add a comment to that effect on top of the variable declaration.